assets/eip-2612/LICENSE

```diff
@@ -0,0 +1,674 @@
+                    GNU GENERAL PUBLIC LICENSE
```

Comment:
I don't think we can allow GPL reference implementations, but let me confer with the other editors.

Reply:
I agree with this. This must be changed.